upup/pkg/fi/nodeup/nodetasks/package.go

```diff
@@ -293,16 +303,30 @@ func (_ *Package) RenderLocal(t *local.LocalTarget, a, e, changes *Package) erro
 			}
 
 			var args []string
+			env := os.Environ()
 			if t.HasTag(tags.TagOSFamilyDebian) {
-				args = []string{"dpkg", "-i"}
+				// Only Debian releases newer than Jessie can install .deb via apt-get
+				// TODO: Refactor this function when Jessie support is dropped (duplicated code)
+				if t.HasTag(tags.TagOSDebianJessie) {
```

Comment:
Is there a way to structure this such that we're conditionally checking for a debian OS prior or equal to Jessie (as opposed to specifically checking for jessie vs other stuff)? I feel like checking if before version, `dpkg` it, but if you're beyond it, do it with `apt-get` would be a bit safer and also make more sense in the future. thoughts?

Reply:
Kops supports very few OS releases, I guess all of them being here:
https://github.com/kubernetes/kops/blob/1.16.0-alpha.2/nodeup/pkg/distros/distribution.go#L27

My thought was that Jessie is the last of the Debian family with such an old version of `apt-get` that did not support installing from local package. Starting next year it will be EOL also.
Pretty much it's Jessie vs everything else.

Reply:
fair enough- you've convinced me :) thanks for putting up with my inquisition(s)! 
my background seems to steer my perspective and trend my comments toward trying to ensure that we don't break anyone's experience. ~~sometimes~~(often) at the expense of being a pain. i think we've covered our bases here and this lgtm

Reply:
I quite enjoy a thorough review.  Looking forward for more. Thanks!